### ISSUE TITLE
Getting click sounds and static sounds on the phone when run.

### ISSUE THREAD

**Issue by cyai** [twilio-labs/call-gpt#18]
I am getting just click sounds and static sounds the one you get when you tap the mic and then nothing.

Any reason why this is happening?

**Comment by Sheldenshi** [comment]
I am getting this too

**Comment by ninjas005** [comment]
I am also getting shot noises instead of speech in my phone. What could be the problem causing this?

**Comment by ninjas005** [comment]
anyone find a solution for this

**Comment by cyai** [comment]
I figured it out. For me it was the issue with the TTS service I was using, ElevenLabs. I ran out of credits there so it was throwing me the error. I wasn't handling the error properly in my code thus the error was being streamed which was leading to the clicking sound. 

Check if the audio you are trying to stream is actually an audio buffer, it might be something else which might lead to the clicking sound
